releasenotes/notes/apm-profile-proxy-720c4e90dee22f33.yaml

```
@@ -0,0 +1,12 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    APM: A new endpoint was added which helps forward and augment profiles
+    to the intake.
```

Nit:

```suggestion
    APM: A new endpoint was added which helps augment and forward profiles
    to Datadog's intake.
```

Sounds a bit more clearer imo.